rts/Sim/MoveTypes/MoveType.cpp

```diff
@@ -42,7 +42,8 @@ AMoveType::AMoveType(CUnit* owner):
 	maxSpeedDef(owner? owner->unitDef->speed / GAME_SPEED : 0.0f),
 	maxWantedSpeed(owner? owner->unitDef->speed / GAME_SPEED : 0.0f),
 
-	maneuverLeash(500.0f)
+	maneuverLeash(500.0f),
+	waterline(owner->unitDef->waterline)
```

Comment:
should be `waterline(owner? owner->unitDef->waterline: 0.0f)` due to the workings of CREG